lua/refactoring/refactor/106.lua

```diff
@@ -64,44 +64,47 @@ local function get_extract_setup_pipeline(bufnr)
         :add_task(get_selected_local_defs)
 end
 
+local function extract_setup(refactor)
+    local selected_local_references = get_selected_local_references(refactor)
+
+    local function_name = get_input("106: Extract Function Name > ")
```

Comment:
btw, there is so much I hate to this function in general :)

But I am waiting to do something about it.  I am positive I'll have to do some very similar operations in the near future.  THEN!! i'll figure out a way to pull things out.

Reply:
That makes sense, you want me to close this and cherry pick the small bug fix for get text to another branch?